mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -1282,6 +1282,10 @@ linux_node_config {
 
 * `vulnerability_mode` - (Optional) Sets the mode of the Kubernetes security posture API's workload vulnerability scanning. Available options include `VULNERABILITY_DISABLED`, `VULNERABILITY_BASIC` and `VULNERABILITY_ENTERPRISE`.
 
+<a name="nested_security_posture_config"></a>The `fleet` block supports:
+
+* `project` - (Optional) The name of the Fleet host project where this cluster will be registered.
+
```

Comment:
This block is the doc to describe the fields in the `fleet` field.  However, `fleet` field is needed to be added to the doc separately. Please see the example https://github.com/GoogleCloudPlatform/magic-modules/blob/6f22d2b46d19565c6fd787d7cf78d0ca027a026c/mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown?plain=1#L378